schemas/constructs/v1beta1/model/model.json

```diff
@@ -122,20 +135,16 @@
       }
     },
     "subCategory": {
-      "type": "string",
-      "description": "Sub-category of the model.",
-      "minLength": 1,
-      "maxLength": 100,
-      "x-oapi-codegen-extra-tags": {
-        "yaml": "subCategory",
-        "json": "subCategory"
-      },
+      "$ref": "../subcategory/subcategory.json",
```

Comment:
add a x-import-go and type so the generated code wont embed it but import the go struct